# HG changeset patch
# User Jaroslav Hajek <highegg@gmail.com>
# Date 1251269731 -7200
#      Wed Aug 26 08:55:31 2009 +0200
# Node ID 9e4333169eae1614b05226cf2eb1eb2381029dbe
# Parent  f5c28d8f5147006b7047dc21614b273330c8fbb8
fix typo in last patch

diff --git a/src/syscalls.cc b/src/syscalls.cc
--- a/src/syscalls.cc
+++ b/src/syscalls.cc
@@ -256,17 +256,17 @@ waitpid (pid);\n\
      @print{} these\n\
      @print{} strings\n\
      @print{} some\n\
      @print{} are\n\
 @end example\n\
 \n\
 Note that @code{popen2}, unlike @code{popen}, will not \"reap\" the\n\
 child process.  If you don't use @code{waitpid} to check the child's\n\
-exit status, it will linger until Octave exits.
+exit status, it will linger until Octave exits.\n\
 @end deftypefn")
 {
   octave_value_list retval;
 
   retval(2) = -1;
   retval(1) = Matrix ();
   retval(0) = Matrix ();
 
